src/core/reference/src/runtime/reference/tile.cpp

```diff
@@ -49,6 +49,12 @@ void runtime::reference::tile(const char* arg,
     std::vector<size_t> indices(in_shape_expanded.size() - 1, 0);
     size_t axis = indices.size();
 
+    if (std::all_of(repeats.begin(), repeats.end(), [](int64_t repeat) {
```

Comment:
@artkuli Could you add new tests?

Reply:
Added test to check Tile with 2D shape and 0 repeats